pyiron/vasp/base.py

```diff
@@ -468,24 +468,18 @@ def collect_errors(self):
         """
         num_eddrmm, snap = self._get_eddrmm_info()
 
+        warning_string = "EDDRMM warnings are ignored. EDDRMM occures {} times, first in ionic step {}."
```

Comment:
shall be "warnings are ignored" be always printed? Even if  `self.get_eddrmm_handling() == "warn"`?

Reply:
Fair. Now it just says "EDDRMM warnings occured {} times, first in ionic step {}"